### ISSUE TITLE
lerna exec/run with ERR! lerna Unknown arguments: include-merged-tags, includeMergedTags

### ISSUE THREAD

**Issue by Aqours** [lerna/lerna#2332]
## Expected Behavior
Works fine as lerna v3.16.4

## Current Behavior
throw ERR! lerna Unknown arguments: include-merged-tags, includeMergedTags

## Steps to Reproduce (for bugs)

1. run `lerna exec --since --include-merged-tags -- pwd` in monorepo
2.
3.
4.

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

## Context
I want run or exec command since merged tags. And `lerna exec --since -- pwd` will exclude other merged tags.

Why I'm not use `lerna exec --since origin/master -- pwd`?
It will throw errors if I add new sub-package on other branch than master, because master does not include new sub-package now.

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.18.3 |
| `npm --version`  | 6.12.0 |
| `yarn --version` | VERSION |
| `node --version` | v12.13.0 |

| OS | Version |
| --- | --- |
| macOS Catalina | 10.15.1 |

**Comment by evocateur** [comment]
`--include-merged-tags` is only supported on [`lerna version`](https://github.com/lerna/lerna/tree/master/commands/version#--include-merged-tags) (and implicitly on `lerna publish` and `lerna changed`). That it ever worked on `lerna exec`/`lerna run` is an accident of history (and never documented as such).

I guess #1712 should have added it to `@lerna/filter-options` as well. Or another method entirely. Or I should just give up on caring about `strict()`. 🤷‍♂

**Comment by evocateur** [comment]
Published in [v3.18.4](https://github.com/lerna/lerna/releases/tag/v3.18.4)
